src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -846,6 +846,15 @@ export class NotebookModel extends Disposable implements INotebookModel {
 			if (newConnection) {
 				if (newConnection.serverCapabilities?.notebookKernelAlias) {
 					this._currentKernelAlias = newConnection.serverCapabilities.notebookKernelAlias;
+					// Removes SQL kernel to Kernel Alias Connection Provider map
```

Comment:
would be useful for the comment to explain why this is being done, rather than just mentioning what is being done.